text/0419-cdk-launchpads.md

```diff
@@ -0,0 +1,539 @@
+
+There are two ways of customizing a Launchpad: using JSON/YAML files, or using CDK code.
+
+#### Customizing the Launchpad with JSON/YAML files
+
+This option of customizing the Launchpad is geared primarily towards platform administrators
+who don't want to use a programming language (and thus, CDK code) to customize their Launchpads.
+Using a JSON or YAML file in a specific format allows customizing all aspects of the Launchpad,
+without the need to write a single line of code.
+
+The JSON file has the basic structure (the YAML file is very similar):
+
+```json
+{
+  "launchpad": {
+    ...
+  }
+}
+```
+
+We show examples of different parts of the JSON file in each of the sections below.
+
+You apply the changes using the JSON/YAML file by passing the file as an argument to the Launchpad commands,
+for example `cdk-lpads update my-launchpad.json/my-launchpad.yaml`.
+
+Deploying the Launchpads through a pipeline will also be supported with
+[CDK Pipelines](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.pipelines-readme.html).
+
+#### Customizing the Launchpad with CDK code
+
+It's also possible to customize the Launchpad using CDK code.
+In that scenario, the Launchpad is basically a CDK application, and you execute the Launchpads CLI commands,
+like `cdk-lpads update`, inside of it, similarly like you execute CDK CLI commands inside your CDK application.
+
+A CDK application for a Launchpad has a very similar structure to a "regular" CDK application,
+but the classes used are slightly different (the example is in TypeScript):
+
+```ts
+import * as lpads from '@aws-cdk/cdk-launchpads';
+
+const launchpad = new lpads.Launchpad();
+new lpads.StandardCdkBootstrap(launchpad, 'CdkBootstrap', {
+  // options to customize the CDK bootstrap can be passed here...
+});
+
+// additional things can be changed in the Launchpad here...
+
+launchpad.synth();
+```
+
+This CDK application will synthesize to the JSON configuration mentioned above,
+similarly like "regular" CDK applications synthesize to CloudFormation templates.
+
+We will show details of customizing the Launchpad using CDK code in each of the sections below.
+
+### Launchpad features
+
+#### Compliance
+
+Compliance rules allow administrators to restrict what kind of AWS resources can be deployed in a given environment.
+They can be used both for security purposes,
+but also for things like checking whether a given application conforms to the AWS Well-Architected principles.
+
+Launchpads allow defining compliance rules using the
+[CFN Guard 2.0 tool](https://github.com/aws-cloudformation/cloudformation-guard).
+Those compliance rules are then deployed as
+[CloudFormation Hooks](https://docs.aws.amazon.com/cloudformation-cli/latest/userguide/hooks.html),
+which means it's impossible to deploy a CDK application into a given environment that violates any compliance rule defined for it.
+
+Compliance rules can be defined in JSON/YAML directly:
```

Comment:
What about being able to provide a path to CFN Guard rules file? It will then allow to develop the rules separately, including capabilities such as [testing](https://docs.aws.amazon.com/cfn-guard/latest/ug/testing-rules.html).

Reply:
Yes, that's a great point! Will add.

Reply:
Done in the latest revision.